pyenv-lazy.plugin.zsh

```diff
@@ -1,5 +1,17 @@
-# Try to find pyenv, if it's not on the path
 export PYENV_ROOT="${PYENV_ROOT:=${HOME}/.pyenv}"
+
+# Try to install pyenv, if $PYENV_ROOT is empty
+function _zsh_pyenv_install() {
+    echo "Installing pyenv..."
+    git clone "https://github.com/pyenv/pyenv.git" "${PYENV_ROOT}"
+    if [[ -n "${ZSH_PYENV_LAZY_VIRTUALENV}" ]]; then
+        echo "Installing pyenv-virtualenv..."
+        git clone "https://github.com/pyenv/pyenv-virtualenv.git" "${PYENV_ROOT}/plugins/pyenv-virtualenv"
+    fi
+}
+[[ ! -d "${PYENV_ROOT}" ]] && _zsh_pyenv_install
```

Comment:
Wouldn't this be a better check, to not try to install when there is a file there?
```suggestion
[[ ! -e "${PYENV_ROOT}" ]] && _zsh_pyenv_install
```